### ISSUE TITLE
FNX-5055 ⁃ [Bug] TaskCluster links to `raptorRelease` build types aren't configured correctly

### ISSUE THREAD

**Issue by ncalexan** [mozilla-mobile/fenix#1478]
## Steps to reproduce
The ARM build at https://tools.taskcluster.net/groups/NpA6M3XySpy-ZSlPUOgEoA/tasks/YRqdzPTfTWyLUUKpHUu_tw/runs/0/artifacts doesn't have a `target.apk` at all.

The aarch64 build at https://tools.taskcluster.net/groups/NpA6M3XySpy-ZSlPUOgEoA/tasks/G-yJwOBQTCmMA_n4yBQE7Q/runs/0/artifacts lists `target.apk` -- the link is to https://queue.taskcluster.net/v1/task/G-yJwOBQTCmMA_n4yBQE7Q/runs/0/artifacts/public/target.apk -- but that produces:

```
{
  "reason": "file-missing-on-worker",
  "message": "Artifact \"public/target.apk\" not found at \"/opt/fenix/app/build/outputs/apk/aarch64Greenfield/releaseraptor/app-aarch64-greenfield-releaseraptor.apk\""
}
```

Now, if I had to guess, some code somewhere isn't expecting mixed-case Gradle build types and is lower-casing the `releaseRaptor` build type (rather than lower-casing the initial letter only).  But I don't have a guess for the first bit; not clear why ARM and aarch64 are different here.

### Expected behavior
Gimme APKs!

### Actual behavior
No APKs, either listed or to fetch.

**Comment by ncalexan** [comment]
Yep, there it is: https://searchfox.org/mozilla-mobile/rev/229ab3755d5caedb02066cd524b2340191ce8309/fenix/automation/taskcluster/lib/tasks.py#482

**Comment by ncalexan** [comment]
> The ARM build at https://tools.taskcluster.net/groups/NpA6M3XySpy-ZSlPUOgEoA/tasks/YRqdzPTfTWyLUUKpHUu_tw/runs/0/artifacts doesn't have a `target.apk` at all.

Good news, everyone!  I looked at the wrong task, and
https://tools.taskcluster.net/groups/NpA6M3XySpy-ZSlPUOgEoA/tasks/EE8tR1nBTIme_lvchwvohQ/runs/0/artifacts does indeed have a `target.apk`.  But the link is broken :)

**Comment by ncalexan** [comment]
This was addressed by #1321.
